R/axis.R

```diff
@@ -14,12 +14,11 @@
 #' @rdname axis
 #' @export
 #' @examples
-#' 
 #' gs <- gsplot() %>%
 #'    points(x=1:5, y=1:5, legend.name="Stuff") %>%
 #'    lines(2:6, y=2:6, ylim=c(0,10)) %>%
 #'    axis(side=c(3,4),labels=FALSE) %>%
-#'    legend(location="topright")
+#'    legend("topright")
```

Comment:
Why did you take that away? I think it's a coincidence that topright works correctly because that is the default in graphics, but we need to include `location` now so that we can also support default graphic legend. So, I'd rather keep all the locations in all examples.